pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -1,124 +1,110 @@
+IsingXY                = { properties = [ "invertible",                 "differentiable" ] }
+IsingYY                = { properties = [ "invertible",                 "differentiable" ] }
+IsingZZ                = { properties = [ "invertible",                 "differentiable" ] }
+MultiRZ                = { properties = [ "invertible",                 "differentiable" ] }
+PauliX                 = { properties = [ "invertible",                 "differentiable" ] }
+PauliY                 = { properties = [ "invertible",                 "differentiable" ] }
+PauliZ                 = { properties = [ "invertible",                 "differentiable" ] }
+PhaseShift             = { properties = [ "invertible",                 "differentiable" ] }
+QubitUnitary           = { properties = [ "invertible",                 "differentiable" ] }
+Rot                    = { properties = [ "invertible",                 "differentiable" ] }
+RX                     = { properties = [ "invertible",                 "differentiable" ] }
+RY                     = { properties = [ "invertible",                 "differentiable" ] }
+RZ                     = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitationMinus  = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitationPlus   = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitation       = { properties = [ "invertible",                 "differentiable" ] }
+S                      = { properties = [ "invertible",                 "differentiable" ] }
+SWAP                   = { properties = [ "invertible",                 "differentiable" ] }
+Toffoli                = { properties = [ "invertible",                 "differentiable" ] }
+T                      = { properties = [ "invertible",                 "differentiable" ] }
 
 # Operators that should be decomposed according to the algorithm used
 # by PennyLane's device API.
 # Optional, since gates not listed in this list will typically be decomposed by
 # default, but can be useful to express a deviation from this device's regular
 # strategy in PennyLane.
-decomp = [
-        "BasisState",
-        "QubitStateVector",
-        "StatePrep",
-        "QFT",
-        "MultiControlledX",
-]
+[operators.gates.decomp]
+
+BasisState             = {}
+MultiControlledX       = {}
+QFT                    = {}
+QubitStateVector       = {}
+StatePrep              = {}
 
 # Gates which should be translated to QubitUnitary
-matrix = [
-        "ControlledQubitUnitary",
-        "ECR",
-        "SX",
-        "ISWAP",
-        "PSWAP",
-        "SISWAP",
-        "SQISW",
-        "CPhase",
-        "OrbitalRotation",
-        "QubitCarry",
-        "QubitSum",
-        "DiagonalQubitUnitary",
-        "BlockEncode",
-]
+[operators.gates.matrix]
+
+BlockEncode            = {}
+CPhase                 = {}
```

Comment:
Is that not `ControlledPhaseShift`?

Reply:
PennyLane has this alias [here](https://github.com/grwlf/pennylane/blob/58ad1dfdc661fb58034225743503167e3006d86f/pennylane/ops/op_math/controlled_ops.py#L2000) so technically it is a valid gate. The purpose of this config is to describe what to do with operations, and this line should instructs it to decompose `CPhase` into a matrix. But we should do the same with `ControlledPhaseShift`, while in the current version we mark it as native. This is strange, I think we need to move `CPhase` to the `native` set as well.

Reply:
This update was forgotten, and I agree that CPhase and ControlledPhaseShift should not be in different operation types. However, the weird part is that `CPhase` is not even its own class, so you would never see one on tape. I think the correct course of action is remove it from the TOML as well as removing it from the device class:
https://github.com/PennyLaneAI/pennylane-lightning/blob/a254c95ad5140c94a51c95f7fafab1b344757c45/pennylane_lightning/lightning_kokkos/lightning_kokkos.py#L116

Reply:
It is not a class name, it is just a local variable name. So this gate doesn't exist!

Reply:
OK, got it, we can remove it from toml schema. Still, there are minor problems: 1) CPhase exists in the toml schema 1 configs 2) CPhase does exists in PL's `device.operations` as a separate gate. You said in other discussion that Ali is addressing (2). Meanwhile I have addressed (1) in  https://github.com/PennyLaneAI/catalyst/pull/718